### PR TITLE
Better landing

### PR DESCRIPTION
**Pull request by SoyOscarRH**
# Abstract
- Header centered in mobile
- More consistent colors
- Use of semantic html
- Better buttons in desktop
- Better layout in mobile

# Before
before
Screen Shot 2020-04-26 at 12 47 57 AM

# After
Screen Shot 2020-04-26 at 12 48 04 AM

ezgif com-optimize

### CODE REVIEW

File: fudge-ratio/src/main/webapp/landing.css [hunk 1]
```diff
@@ -1,99 +1,103 @@
...
   --accent-color: rgba(45, 93, 122, 0.75);
   --font-color: rgb(37, 51, 51);
-  --base: min(3.5vw, 1.7rem);
-  --font-base: calc(1.2vw + 0.8rem);
+  --font-base: calc(0.8vw + 0.6rem);
 }
 
 html {
-  background: var(--accent-color) !important;
+  background: var(--background-color);
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/landing.css]: no important, better code

File: fudge-ratio/src/main/webapp/landing.css [hunk 2]
```diff
@@ -1,99 +1,103 @@
...
-#logo {
-  margin: auto auto;
-  padding-top: calc(0.6 * var(--font-base));
-  word-wrap: break-word;
-  font-size: calc(2 * var(--font-base));
-  color: #f2f2f2;
-}
-
-.banner {
+header {
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/landing.css]: so much clearer, talking about header and main

File: fudge-ratio/src/main/webapp/landing.css [hunk 3]
```diff
@@ -1,99 +1,103 @@
...
 }
 
-.textBox {
+main {
+  min-height: 80vh;
   background: var(--background-color);
   padding: 15vh 4vw 10vh 4vw;
-  font-size: min(1.8rem, calc(1.25 * var(--font-base)));
-  font-family: 'Open Sans', sans-serif;
+  font-size: calc(1.1 * var(--font-base));
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/landing.css]: simpler

**Comment by SoyOscarRH** [comment]
See the color now ❤️

File: fudge-ratio/src/main/webapp/about/index.html [hunk 4]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <meta name="theme-color" content="#5D7F95">
```
Comment (yaaowusu00) [inline fudge-ratio/src/main/webapp/about/index.html:5]: I didn't even realize we could do this! This is cool

File: fudge-ratio/src/main/webapp/activitiesManager.html [hunk 5]
```diff
@@ -10,12 +10,15 @@ <h1>Add New Activities</h1>
...
-    </form>
+            <input type="text" placeholder="Activity Name" name="name"></input>
+            <input type="number" placeholder="Time Taken" name="newTime"></input>
+            <input type="number" placeholder="Expected Time" name="expectedTime"></input>
+            <input type="text" placeholder="Type" name="type"></input>
+            <br/>
+            <button>Submit</button>
+        </form>
+        <form action="/setup" method="post">
+            <input type="submit" value="Load Data" />
```
Comment (yaaowusu00) [inline fudge-ratio/src/main/webapp/activitiesManager.html:21]: I was going to move the button too, thanks for doing this!

File: fudge-ratio/src/main/webapp/index.html [hunk 6]
```diff
@@ -3,7 +3,9 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8" />
         <meta name="theme-color" content="#5D7F95">
-        <link rel="stylesheet" href="landing.css" />
+        <link rel="stylesheet" href="styles/general.css" />
+        <link rel="stylesheet" href="styles/header.css" />
```
Comment (yaaowusu00) [inline fudge-ratio/src/main/webapp/index.html:7]: Much cleaner, thanks!

**Review by yaaowusu00 — APPROVED** [review]
I really like all these changes. I really appreciate when you make additions to my code, I always learn something new from you!

**Pull request merged**